spec/ttml1.xml

```diff
@@ -7070,20 +7070,20 @@ set of <emph>E</emph>, <emph>SSS(E)</emph>;</p></item>
 <item>
 <p><phrase role="strong">[implicit inheritance]</phrase> if the
```

Comment:
I would prefer as a matter of style not to mix the setting of initial values of non-inheritable style properties into the implicit inheritance steps. Rather, I think the resolution to this issue would be better as a new step 7, such as [apply initial property values] along the lines of:

_7. **[apply initial non-inheritable style property values]** if the element type of E is not the styling element type `style`, then for each non-inheritable style property P in the set of style properties defined above in 8.2 Styling Attribute Vocabulary, perform the following ordered sub-steps:
     a. if P is present in the specified style set of E, SSS(E), then continue to the next style property;
     b. if P applies to E, then set P′ to the initial value of property P, where the initial value of a property is determined according to the specific property definition found above in 8.2 Styling Attribute Vocabulary, and is undefined if no initial value is defined;
     c. if the value of P′ is not undefined, then merge P′ into the specified style set of E, SSS(E)._

Reply:
@nigelmegitt This would not match TTML2 ED. I think we should keep this structure, and file a separate ticket to consider updating both TTML2 and TTML1 to match your suggested structure.

Reply:
OK I hadn't spotted that @palemieux , happy to go with your suggestion. Thanks for the pointer.